lib/Targets/AIETargetAirbin.cpp

```diff
@@ -81,7 +81,7 @@ static constexpr auto ME_SS_SLAVE_SLOT_BASE = 0x3F200u;
 /*
   Tile DMA
 */
-static constexpr auto ME_DMA_BD_COUNT = 16;
+static constexpr auto ME_DMA_BD_COUNT = 48;
```

Comment:
This should still be 16 since this target only considers AIE not AIE-ML. Better yet should be using the Device Models.

Reply:
I'm just gonna change it back - that thing is coupled to a bunch of `static_assert`s and statically sized arrays. cc @jnider for when you come back to airbin...